src/language/from_relay/mod.rs

```diff
@@ -809,6 +847,29 @@ fn compile_expression(
                         compute(glenside_expr, ComputeType::ReLU, data_id)
                     }
                 }
```

Comment:
Can you add tests for all of the new Relay constructs we support here? I recognize that some of them we can't currently test because codegen is not implemented for them -- these you can just mark with an `#[ignore = "..."]` (see comment above). But some of these can be tested, like concatenate, reshape, split, transpose, and squeeze (there might be others).

Reply:
added tests for all the new Relay constructs added in this PR! i commented out those that fail because of the issue with using #[ignore]